Lesson_1.py

```diff
@@ -29,16 +37,25 @@ def task_one(list_of_drugs):
 print("\n")
     
 # Write a function that takes in a list of drugs, returns all unique 2-drug combinations as a list of pairs
-def task_two(list_of_combinations):
-    for idx, drug in enumerate(list_of_combinations):
-        if drug[0] == drug[1]:
-            list_of_combinations.pop(idx)
-    return list_of_combinations 
-
-unique_combinations = task_two(all_combinations)
+def make_unique_combinations(list_of_items: list) -> list:
+    """
+    Take list of items and return all unique 2-item combinations 
+    
+    Args:
+    list_of_items (list): a list of items to be filtered for unique combinations
+
+    Returns:
+    list: A list of lists of unique pairs
+    """
+    for idx, drug in enumerate(list_of_items):
```

Comment:
I don't think we need enumerate here or pop. We have a list so we can just [remove](https://www.programiz.com/python-programming/methods/list/remove) the item. Or if you are returning a new list, then just not append it.

Reply:
Or implement option 1